sql/migrations/20230903173317_world.sql

```diff
@@ -0,0 +1,134 @@
+INSERT INTO `spell_threat` (`entry`, `Threat`, `multiplier`, `ap_bonus`) VALUES (19264, 40, 0, 0);
+-- Touch of Weakness R5
+INSERT INTO `spell_threat` (`entry`, `Threat`, `multiplier`, `ap_bonus`) VALUES (19265, 50, 0, 0);
+-- Touch of Weakness R6
+INSERT INTO `spell_threat` (`entry`, `Threat`, `multiplier`, `ap_bonus`) VALUES (19266, 60, 0, 0);
+
+-- Nature's Grasp R1
+INSERT INTO `spell_threat` (`entry`, `Threat`, `multiplier`, `ap_bonus`) VALUES (16689, 10, 0, 0);
+-- Nature's Grasp R2
+INSERT INTO `spell_threat` (`entry`, `Threat`, `multiplier`, `ap_bonus`) VALUES (16810, 18, 0, 0);
+-- Nature's Grasp R3
+INSERT INTO `spell_threat` (`entry`, `Threat`, `multiplier`, `ap_bonus`) VALUES (16811, 28, 0, 0);
+-- Nature's Grasp R4
+INSERT INTO `spell_threat` (`entry`, `Threat`, `multiplier`, `ap_bonus`) VALUES (16812, 38, 0, 0);
+-- Nature's Grasp R5
+INSERT INTO `spell_threat` (`entry`, `Threat`, `multiplier`, `ap_bonus`) VALUES (16813, 48, 0, 0);
+-- Nature's Grasp R6
+INSERT INTO `spell_threat` (`entry`, `Threat`, `multiplier`, `ap_bonus`) VALUES (17329, 58, 0, 0);
+
+-- Nature's Grasp Entangling Roots proc R1
+INSERT INTO `spell_threat` (`entry`, `Threat`, `multiplier`, `ap_bonus`) VALUES (19975, 16, 1, 0);
+-- Nature's Grasp Entangling Roots proc R2
+INSERT INTO `spell_threat` (`entry`, `Threat`, `multiplier`, `ap_bonus`) VALUES (19974, 36, 1, 0);
+-- Nature's Grasp Entangling Roots proc R3
+INSERT INTO `spell_threat` (`entry`, `Threat`, `multiplier`, `ap_bonus`) VALUES (19973, 56, 1, 0);
+-- Nature's Grasp Entangling Roots proc R4
+INSERT INTO `spell_threat` (`entry`, `Threat`, `multiplier`, `ap_bonus`) VALUES (19972, 76, 1, 0);
+-- Nature's Grasp Entangling Roots proc R5
+INSERT INTO `spell_threat` (`entry`, `Threat`, `multiplier`, `ap_bonus`) VALUES (19971, 96, 1, 0);
+-- Nature's Grasp Entangling Roots proc R6
+INSERT INTO `spell_threat` (`entry`, `Threat`, `multiplier`, `ap_bonus`) VALUES (19970, 116, 1, 0);
+
+-- Fear Ward 
+INSERT INTO `spell_threat` (`entry`, `Threat`, `multiplier`, `ap_bonus`) VALUES (6346, 20, 0, 0);
+
+-- Touch of Weakness proc R1
+INSERT INTO `spell_threat` (`entry`, `Threat`, `multiplier`, `ap_bonus`) VALUES (2943, 20, 1, 0);
+-- Touch of Weakness proc R2
+INSERT INTO `spell_threat` (`entry`, `Threat`, `multiplier`, `ap_bonus`) VALUES (19249, 40, 1, 0);
+-- Touch of Weakness proc R3
+INSERT INTO `spell_threat` (`entry`, `Threat`, `multiplier`, `ap_bonus`) VALUES (19251, 60, 1, 0);
+-- Touch of Weakness proc R4
+INSERT INTO `spell_threat` (`entry`, `Threat`, `multiplier`, `ap_bonus`) VALUES (19252, 80, 1, 0);
+-- Touch of Weakness proc R5
+INSERT INTO `spell_threat` (`entry`, `Threat`, `multiplier`, `ap_bonus`) VALUES (19253, 100, 1, 0);
+-- Touch of Weakness proc R6
+INSERT INTO `spell_threat` (`entry`, `Threat`, `multiplier`, `ap_bonus`) VALUES (19254, 120, 1, 0);
+
+-- Remove Blessing of Wisdom
+-- It had 0 threat value and should not exist in spell_threat
+DELETE FROM `spell_threat` WHERE  `entry`=19742;
```

Comment:
```
World of Warcraft Client Patch 1.9.0 (2006-01-03)
- Blessing of Wisdom - Mana regeneration over time will no longer
  generate threat.
  ```

Have you tested if it still generates threat pre 1.9 with this deletion? We should not impose the 1.12 state on previous patches.

Reply:
With this PR and without this PR Blessing of Wisdom generates 0 threat when cast.
The patch note you mention is not currently implemented in vmangos, mp5 from Blessing of Wisdom is currently not able to cause threat in 1.8.4 or below.